runelite-client/src/main/java/net/runelite/client/plugins/skillcalculator/SkillCalculator.java

```diff
@@ -155,19 +156,39 @@ private void clearCombinedSlots()
 
 	private void renderBonusOptions()
 	{
-		if (skillData.bonuses != null)
+		if (skillData.getBonuses() != null)
 		{
-			for (SkillDataBonus bonus : skillData.bonuses)
+			for (SkillDataBonus bonus : skillData.getBonuses())
 			{
 				JPanel uiOption = new JPanel(new BorderLayout());
-				JLabel uiLabel = new JLabel(bonus.name);
+				JLabel uiLabel = new JLabel(bonus.getName());
 				JCheckBox uiCheckbox = new JCheckBox();
 
 				// Adding an empty 8-pixel border on the left gives us nice padding.
 				uiOption.setBorder(BorderFactory.createEmptyBorder(0, 8, 0, 0));
 
 				// Adjust XP bonus depending on check-state of the boxes.
-				uiCheckbox.addActionListener(e -> adjustXPBonus(uiCheckbox.isSelected(), bonus.value));
+				uiCheckbox.addActionListener(e -> adjustXPBonus(uiCheckbox.isSelected(), bonus.getValue()));
+
+				uiOption.add(uiLabel, BorderLayout.WEST);
+				uiOption.add(uiCheckbox, BorderLayout.EAST);
+				add(uiOption);
+			}
+		}
+
+		if (skillData.getMultipliers() != null)
```

Comment:
Hmm actually, why is this completely different list? i would rather use delimiting boolean probably.

Reply:
```java
private void adjustXPBonus(boolean addBonus, float value)
	{
		xpFactor += addBonus ? value : -value;
		calculate();
	}

	private void adjustXPMultiplier (boolean addMultiplier, float value)
	{
		xpFactor = addMultiplier ? value : 1.0f;
		calculate();
	}
```

Reply:
@deathbeam 
Skilling outfits add a percentage to base xp
Gilded Altar and Ecto are a percentage of base xp

This needed to be seperate in the lists and idendtifiable.

Reply:
adjustXpBonus is also the original code.

adjustXpMultiplier is the addition so it only affects the prayer calculator.